index.js

```diff
@@ -25,6 +25,7 @@ app.get('/', (req, res) => {
 // Endpoint to handle questions
 app.post('/ask', async (req, res) => {
     try {
+        console.log('sending request');
```

Comment:
Makes sense to add a log here 💯 💯  Do you think you can make this more descriptive?